weaver/core/drivers/fabric-driver/server/satp.ts

```diff
@@ -0,0 +1,363 @@
+import satp_pb from '@hyperledger/cacti-weaver-protos-js/relay/satp_pb';
+import satp_grpc_pb from '@hyperledger/cacti-weaver-protos-js/relay/satp_grpc_pb';
+import driverPb from '@hyperledger/cacti-weaver-protos-js/driver/driver_pb';
+import logger from './logger';
+import { credentials } from '@grpc/grpc-js';
+import { SatpAssetManager, HashFunctions } from '@hyperledger/cacti-weaver-sdk-fabric'
+import fs from 'fs';
+import path from 'path';
+import { Gateway, Network, Wallets, Contract, X509Identity } from 'fabric-network'
+import { getNetworkGateway } from "./fabric-code";
+import { getDriverKeyCert } from './walletSetup';
+
+const DB_NAME: string = "driverdb";
+const DRIVER_ERROR_CONSTANTS = JSON.parse(
+    fs.readFileSync(
+        path.resolve(__dirname, '../constants/driver-error-constants.json'),
```

Comment:
It may break in published package (unless you copy the .json to dist/ folder manually). `import` json file directly at build time (can be allowed in typescript config, done for other cacti packages)

Reply:
Good point! I'd recommend using a Typescript import with JSON assertions. We are leveraging the same thing in other places in the code to pull in openapi.json spec files for example.
It's extra nice that this way the Typescript compiler is holding your hand while you are typing - there are auto-complete suggestions for the content within the JSON file that you imported.

Reply:
@outSH This entire module is being published as a Docker image, and not as an NPM package (this is a standalone module, which other packages don't import). The referenced file (with the relative path) will lie in the container at the expected location, so the code won't break unless somebody tries a variation without understanding how the module works.

So I think we should keep this code as is.

Reply:
npm package must not assume runtime environment, if we want to use it outside of container in the future we'll have to spend twice as much time to identify this problem again and fix it then. The change recommended by peter if fairly trivial

Reply:
I was following the same pattern which is available in `events.ts`. Anyway, I just found out that I am not using that constant so I removed it.